tests/acceptance/test_giving_feedback.py

```diff
@@ -0,0 +1,63 @@
+import time
+
+import pytest
+import uvicorn
+
+from multiprocessing.context import Process
+from application.settings import get_settings
+
+settings = get_settings()
+
+settings.READ_DATABASE_URL = (
```

Comment:
Should this be `settings.READ_DATABASE_URL = settings.READ_DATABASE_URL || ... `

Reply:
I think this is overwriting the read database URL here so we wouldn't want to  use `settings.READ_DATABASE_URL = settings.READ_DATABASE_URL || ...`